service_container/tags.rst

```diff
@@ -82,6 +84,16 @@ that it implements ``Twig_ExtensionInterface``) and adds the tag for you.
             }
         }
 
+    If you do not want to create a custom kernel, it is also possible to use `_instanceof`:
+
+    .. code-block:: yaml
```

Comment:
Missing the xml and php documentation format. However, I'm unsure of their notations at this point.

Reply:
For XML: e.g. https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/Tests/Fixtures/xml/services_instanceof.xml
For PHP: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/Tests/Compiler/ResolveInstanceofConditionalsPassTest.php

Reply:
Note that for PHP, setInstanceofConditionals should be called for every single definition, since there is no "file" scope context there